deploy/crds/kieapp.crd.v1beta1.yaml

```diff
@@ -706,7 +706,7 @@ spec:
                     properties:
                       url:
                         type: string
-                        description: LDAP Endpoint to connect for authentication
+                        description: https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/362
```

Comment:
Are you sure?

Reply:
ooops, ctrlC and V problem LOL
thanks for finding this.